rubocop-must.yml

```diff
@@ -425,3 +425,6 @@ Style/SymbolProc:
 
 Style/UnneededInterpolation:
   Enabled: false
+
+Style/FrozenStringLiteralComment:
```

Comment:
Added this since rubocop complained vor me. See https://github.com/bbatsov/rubocop/issues/3284